File: benchmark/results/java-invocation-inline/run-002/b-baseline/output/com/metaengine/demo/billing/value_objects/InvoiceLine.java
package com.metaengine.demo.billing.value_objects;

import java.time.Instant;

/** InvoiceLine value object for the billing domain. */
public record InvoiceLine(
        String id,
        Instant createdAt,
        Instant updatedAt,
        String name,
        String description) {
}

File: benchmark/results/python-multilang/run-002/b-baseline/output/metaengine_demo/billing/value_objects/invoice_line.py
"""InvoiceLine value object for the billing domain."""

from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class InvoiceLine:
    """Immutable single line within an invoice."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str
